### PR TITLE
ET-70: Updating README based on QA's feedback

### PR DESCRIPTION
**Pull request by JakeBat**
# Problem
The README addition for "User Analytics" reused alot from the other SDKs despite a substantial difference in event names and how the hook is used.

# Solution
Updated the README to reflect the uniqueness of the JS SDKs events

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-939-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-939-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-939-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-939-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -570,24 +570,21 @@ Currently we recommend using the above hook to keep track of how many user's rea
...
-VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness video capture", challenge_type can be found in eventProperties 
-VIDEO_FACIAL_CAPTURE_CONFIRMATION - User reached the "liveness video confirmation" screen 
-VIDEO_FACIAL_UPLOAD - User's liveness video is uploading
+screen_welcome - User reached the "Welcome" screen 
+screen_document_front_capture_file_upload - User reached the "document capture" screen for the front side (for two-sided or one-sided document) 
+screen_document_back_capture_file_upload - User reached the "document capture" screen for the back side (for two-sided document) 
+screen_document_front_confirmation - User reached the "document confirmation" screen for the front side (for two-sided or one-sided document) 
+screen_document_back_confirmation - User reached the "document confirmation" screen for the back side (for two-sided document) 
+Starting_upload - User's file is uploading 
+Completed upload - User's file uploaded successfully
```
Comment (kopijunkie) [inline README.md]: Not a blocker for this PR but I initially thought they might've been typos...until I realised this is how it is our code. 😅
We should fix this to be consistent with the other event names IMO @stefaniacardenas thoughts?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://939-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by JakeBat** [comment]
Hey @kopijunkie and @bartzet,

Thanks for taking the time to review the README changes. However after a short discussion with See Wah, it was decided the existing README was correct and I need to add some mapping for the existing event names to match whats described in the existing README when an event is exposed to the public. Apologies for any confusion, but I'll be closing this PR and opening a new one with the correct changes sometime this afternoon my time.

**Pull request closed without merging**